### ISSUE TITLE
Suggestions for customizing the product categories displayed on the home screen

### ISSUE THREAD

**Issue by ysknsid25** [jsdecena/laracom#294]
Good evening @jsdecena 

# TL;DR

I have two suggestions for the display on the home screen.

1. the product categories displayed on the home page
2. product images that have not yet been registered

# 1. the product categories displayed on the home page

Read the code below.

https://github.com/jsdecena/laracom/blob/91602fc8687cb7b4d6038aeead126c234f99cfea/project/app/Http/Controllers/Front/HomeController.php#L27-L43

## Actual

The categories of products displayed on the home screen have been hard-coded with IDs 2 and 3.
This means that users cannot display the newly added categories on the home page.

## Want to

On the administration screen, you can select one or more (up to about 3) product categories to be displayed on the home screen.
(If at least one is not selected, not a single product will be displayed on the home page.)
The home screen should then display the products associated with the category selected by the user.

# 2. product images that have not yet been registered

## Actual

The image does not look right when it is not registered

![image](https://user-images.githubusercontent.com/44870505/215729205-a6569b32-4041-4202-89c2-5c7fcd80b0c3.png)

## Want to

If no image is available, we want to display an image like No Data.

like this

![image](https://user-images.githubusercontent.com/44870505/215729610-b2b99e4e-18bb-4d43-83c1-7b58050f0f38.png)

# Finally

Can we try to modify the above two points in the direction suggested?
Thank you for your confirmation.

**Comment by jsdecena** [comment]
yeah @ysknsid25 these are good suggestions. I am trying to do that before but just found no time but it is always good if we get a help from community. Thanks

**Comment by ysknsid25** [comment]
hi, @jsdecena 

Thanks for the confirmation.
I am glad I could be of assistance.

We are planning to split the implementation into two parts.
I will send pull requests in the order of 2 -> 1.

See you soon.
